electrode_models/conversion_electrode.py

```diff
@@ -1,114 +1,188 @@
 """
-    conversion_electrode.py
+    sulfur_electrode.py
 
-    Class file for conversion electrode methods. An example is lithium-sulfur, where solid S8 phase is converted to Li2S during discharge.
-
-    THe model assumes some intert, electrically conductive host phase (e.g. carbon), plus an arbitrary number of "conversion" phases.  Each conversion phase can possibly participate in reactions at the host/electrolyte interface, plus at their own electrolyte interface.
-
-    Three phase boundaries are not currently implemented, but certainly could be, at some later date.
+    Class file for air electrode methods
```

Comment:
Again, I think my previous change should supersede this.  I'm not sure where these diverged in commit history, but I am guessing, sadly, that a simple rebase would not fix it.  But it is worth a try.

Reply:
Here is the comment that used to be here:

>    Class file for conversion electrode methods. An example is lithium-sulfur, where solid S8 phase is converted to Li2S during discharge.
The model assumes some intert, electrically conductive host phase (e.g. carbon), plus an arbitrary number of "conversion" phases.  Each conversion phase can possibly participate in reactions at the host/electrolyte interface, plus at their own electrolyte interface.
Three phase boundaries are not currently implemented, but certainly could be, at some later date.

Something like this would be good, but I think that last sentence is no longer accurate--is that correct?

Reply:
Correct, three phase boundaries are included now.